### DISCUSSION TITLE
Start Without Debugging

### DISCUSSION THREAD

**Discussion by ADefWebserver** [ADefWebserver/ADefWebserver.Module.HtmlTextV2#18] [General]
@thabaum - To get things to run I have to now start Oqtane in Debug F5 Mode.

![image](https://github.com/ADefWebserver/ADefWebserver.Module.HtmlTextV2/assets/1857799/bdb0c1b4-d6b7-4cde-bd3e-9633f870d564)

When I don't I get:

![image](https://github.com/ADefWebserver/ADefWebserver.Module.HtmlTextV2/assets/1857799/d13884a6-4072-4b52-aa12-891c1f88b991)

**Comment by thabaum** [comment]
I could only get it to run in debug mode as well, I have not had release work yet as something is wrong with the package project.

I will review all the files I have in my project again as I don't copy all my changes as some are experimental but I was trying to get the source editor to you to review.

**Reply by ADefWebserver** [reply]
> I have not had release work yet as something is wrong with the package project.

Got it, understood 👍🏽

**Reply by thabaum** [reply]
the Editor.razor is what we need to focus on now to get it to work with HtmlEditor.Blazor component and then integrating the file manager.a

I was just going to leave everything as is as far as the tab panels go and make it work in both areas.  Just replace the quill editor more or less for now not worried about the other parts until the part is done of saving/editing content.

**Reply by ADefWebserver** [reply]
> I was just going to leave everything as is as far as the tab panels go and make it work in both areas. Just replace the quill editor more or less for now not worried about the other parts until the part is done of saving/editing content.

Sounds good.
